DeploymentExperiences.md

```diff
@@ -0,0 +1,22 @@
+## Honestly I have always struggled a bit with deployment. Especially deploying different DB's is always a little hard for me. Some things that went well:
+
+ - Heroku CLI tools installed very nicely using the ubuntu installation in the documentation
+ - The guide to deploying was actually pretty helpful as well and I appreciated the links to documentation.
+ - Seemed to be easy gettiung the project created using the Heroki CLI interface and connected to my repository.
+ 
+
+## What went poorly:
+
+ - Unbale to run migrations on heroku app and keeps failing
+ - Structure of my app I beleive is throwing off heroku
+ - If I grasped what was happening in the Djorg app a little better I might be able to solve the issue.
+
+
+## Left to do:
+
+ - App is not running and failing to build on heroku. Need to implement.
+ - Need to pay close attention to the solution lecture and try to emulate deployment step by step.
+ - 
+ 
+
+## All in all a very interesting experience and something I have not done with a django project before. Going to take some time and perhaps try to emulate it with docker since I'm familiar with that process.
```

Comment:
Deployment and Heroku can definitely be tricky beasts. It's important to get deployed, so if you're still having issues definitely reach out to us for help.